src/App.js

```diff
@@ -0,0 +1,74 @@
+import { nanoid } from "nanoid";
+import { useState } from "react";
+import { AiFillPlusCircle } from "react-icons/ai";
+import TaskList from "./components/ui/TaskList";
+const App = () => {
+  const [task, setTask] = useState("");
+  const [toDo, setAllToDo] = useState([]);
+
+  const handleOnChange = (event) => {
+    setTask(event.target.value);
+  };
+  const handleTasksubmit = (event) => {
+    if (task !== "") {
+      const newTask = { id: nanoid(), taskName: task, isTaskDone: false };
+      event.preventDefault();
+
+      setAllToDo((prevToDoList) => [...prevToDoList, newTask]);
+      setTask("");
+    }
+  };
+
+  const handleTaskDone = (id) => {
+    console.log(id);
+    setAllToDo((prevToDoList) =>
+      prevToDoList.map((task) =>
```

Comment:
You can remove these brackets on the task argument, you don't need these brackets whenever you have a single argument, it may seem not a big deal but in order to achieve a good readable code, you have also to consider small things.